packages/components/src/mobile/picker/index.ios.js

```diff
@@ -11,12 +11,13 @@ import { Component } from '@wordpress/element';
 
 class Picker extends Component {
 	presentPicker() {
-		const { options, onChange } = this.props;
+		const { options, onChange, title } = this.props;
```

Comment:
Nice addition there 👍

Reply:
Thanks, glad the code accommodated thomas's request here very easily